kod/object/active/holder/nomoveon/battler/monster/towns/Goad.kod

```diff
@@ -442,18 +445,10 @@ messages:
        return;
     }
 
-    FightIsDraw()
```

Comment:
We remove this from Goad because nothing calls it.  Its not possible to end a fight in Goads with a draw right now.